db/migrate/20200430072846_create_invited_users.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateInvitedUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :invited_users do |t|
+      t.integer :user_id
+      t.integer :invite_id
```

Comment:
```suggestion
      t.integer :user_id
      t.integer :invite_id
```

When both columns are not null, do we need to ensure uniqueness?

Reply:
Since only one column (`invite_id`) is not null we can't check for uniqueness.